### DISCUSSION TITLE
GET parameters?

### DISCUSSION THREAD

**Discussion by RayFB-13** [reflex-dev/reflex#433] [Q&A]
I made a simple ID finder to look up an id on an API I found and I need to communicate with a different page but I can't use GET params as I would do with something like PHP. Any way I could do this with GET params? 
`localhost:3000/get`**`?id=102`**

**Comment by Lendemor** [comment]
`State.get_query_params()` will return you a dict of the parameters (the dict contains both URI parameters and GET parameters)

Alternatively you should be able to access the item directly from `State.id`.

**Reply by RayFB-13** [reply]
Is This How You Use It?
![image](https://user-images.githubusercontent.com/74745745/216665204-289ea4ec-4fa4-44d4-a9cd-34ff18dc3d1c.png)
It's sadly not working

**Reply by Lendemor** [reply]
Use `self.get_query_params()` within an event handler.

It is not a classmethod so you have to use it with the State instance.

**Reply by RayFB-13** [reply]
Thanks for telling me
